index.js

```diff
@@ -0,0 +1,24 @@
+const express = require('express')
+const app = express()
+const axios = require('axios')
+app.use(express.json())
+
+
+
+app.post('/stocks/addStock', function(req, res){
```

Comment:
Consider making the path name shorter here. I feel like the path '/stocks/addStock' is a bit long.